en/contributing/build.md

```diff
@@ -160,13 +166,13 @@ make android install
 
 To build for real iOS devices on macOS:
 
-```
+```sh
```

Comment:
Has this ios build been re-tested/changed?

Reply:
Yes.